python/google/protobuf/json_format.py

```diff
@@ -104,7 +104,7 @@ def MessageToJson(
     sort_keys=False,
     use_integers_for_enums=False,
     descriptor_pool=None,
-    float_precision=None):
+    float_precision=8):
```

Comment:
I wonder if this can break existing clients. 

CC @anandolee

Reply:
Yes, it may break some clients. It should be in release note 
In google3, this change did not break much (I do not quit remember, like 2-3 breaks)